Mono.Addins/Mono.Addins.Database/AddinFileSystemExtension.cs

```diff
@@ -106,9 +106,10 @@ public virtual System.Collections.Generic.IEnumerable<string> GetFiles (string p
 		/// <param name='path'>
 		/// The directory
 		/// </param>
-		public virtual System.Collections.Generic.IEnumerable<string> GetDirectories (string path)
+		/// <param name='recursive'>Recursively get subdirectories</param>
+		public virtual System.Collections.Generic.IEnumerable<string> GetDirectories (string path, bool recursive)
 		{
-			return Directory.EnumerateDirectories (path);
```

Comment:
@slluis what do you think about this change? It would help with the number of IO syscalls done by just replacing it with one